server/keyspace/keyspace.go

```diff
@@ -55,7 +55,29 @@ type CreateKeyspaceRequest struct {
 	// Using an existing name will result in error.
 	Name   string
 	Config map[string]string
-	Now    time.Time
+	// Now is the timestamp used to record creation time.
+	Now int64
+}
+
+// NewCreateKeyspaceRequest returns a create keyspace request with necessary arguments.
+// name is the name of the keyspace to be created; config is its initial config; now is the current timestamp.
+func NewCreateKeyspaceRequest(name string, config map[string]string, now int64) *CreateKeyspaceRequest {
+	return &CreateKeyspaceRequest{Name: name, Config: config, Now: now}
+}
+
+// GetName returns the name of the create keyspace request.
```

Comment:
I think the function has already been generated by protobuf

Reply:
No, the createKeyspace is only used by HTTP API, thus is not generated by ProtoBuf

Reply:
My bad, then it doesn't matter if wrap it with a function.